### ISSUE TITLE
Add Federal EnviroMapper layer

### ISSUE THREAD

**Issue by AHolder1** [CAWaterBoardDataCenter/tribal-water-data-map-manual#6]
Add [Federal EnviroMapper layer](https://geopub.epa.gov/myem/efmap/index.html?ve=13,47.236778259277344,-122.35669708251953&pText=Fife,%20WA), US EPA Google Earth watersets
[EnviroMapper for Envirofacts | US EPA](https://enviro.epa.gov/enviro/em4ef.home) Home page

**Comment by Lbenton114** [comment]
US EPA Water Discharges (NPDES)

https://geopub.epa.gov/myem/efmap/index.html?ve=13,47.236778259277344,-122.35669708251953&pText=Fife,%20WA

**Comment by Lbenton114** [comment]
- [ ] need to be made public by GIS Team

**Comment by AHolder1** [comment]
https://www.epa.gov/enviro/about-data

**Comment by dalyw** [comment]
Makes sense! I'm back online and will take a look at this - also NPDES discharge locations.

**Comment by dalyw** [comment]
Working on summary here: https://docs.google.com/spreadsheets/d/1r20LKCo4fps8SI0csguVPEq-dkEZymFDvvC5c0_CXCI/edit?usp=sharing
